src/compile/data.ts

```diff
@@ -287,13 +298,11 @@ export namespace formatParse {
     // if there is no parse needed
     model.forEach(function(fieldDef: FieldDef) {
       if (fieldDef.type === TEMPORAL) {
-        parseComponent = parseComponent || {};
```

Comment:
Yep.  But before this PR, a part of `source` assemble code assumes that `formatParse` component either contains something or is null.    I'm not saying that this is worse, but need to make sure that this doesn't break those parts.